clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/AvroGenericDaVinciClient.java

```diff
@@ -700,6 +700,11 @@ private VeniceConfigLoader buildVeniceConfig() {
       kafkaBootstrapServers = backendConfig.getString(KAFKA_BOOTSTRAP_SERVERS);
     }
 
+    String recordTransformerOutputValueSchema = "null";
+    if (daVinciConfig.isRecordTransformerEnabled() && recordTransformerConfig.getOutputValueSchema() != null) {
+      recordTransformerOutputValueSchema = recordTransformerConfig.getOutputValueSchema().toString();
```

Comment:
nit: This is more personal taste, but:

recordTransformerOutputValueSchema = Objects.toString(recordTransformerConfig.getOutputValueSchema(), null);

This just ties the intention into the line as opposed to expanding the above if statement.

You can optionally take this, it's just a personal style preference.